### ISSUE TITLE
java fatal error

### ISSUE THREAD

**Issue by robot110011** [SonarSource/docker-sonarqube#252]
After running scanner successfully:

INFO: ------------------------------------------------------------------------
INFO: EXECUTION SUCCESS
INFO: ------------------------------------------------------------------------
INFO: Total time: 13.691s
INFO: Final Memory: 8M/179M
INFO: ------------------------------------------------------------------------

I get this error on sonarqube server:
```
(..)
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist analysis | status=SUCCESS | time=3ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist analysis properties | status=SUCCESS | time=10ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist measures | inserts=54 | status=SUCCESS | time=19ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist live measures | insertsOrUpdates=841 | status=SUCCESS | time=425ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist new ad hoc Rules | status=SUCCESS | time=1ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist issues | inserts=54 | updates=0 | merged=0 | untouched=0 | status=SUCCESS | time=82ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist project links | status=SUCCESS | time=9ms
2019.03.01 21:59:19 INFO  ce[AWk7RLsxPqexW6Ftis7E][o.s.c.t.s.ComputationStepExecutor] Persist events | status=SUCCESS | time=4ms
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000000001fc6, pid=252, tid=0x00007f16ee1fa700
#
# JRE version: OpenJDK Runtime Environment (8.0_181-b13) (build 1.8.0_181-8u181-b13-2~deb9u1-b13)
# Java VM: OpenJDK 64-Bit Server VM (25.181-b13 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  0x0000000000001fc6
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /opt/sonarqube/hs_err_pid252.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
2019.03.01 21:59:20 WARN  app[][o.s.a.p.AbstractProcessMonitor] Process exited with exit value [ce]: 134
2019.03.01 21:59:20 INFO  app[][o.s.a.SchedulerImpl] Process [ce] is stopped
2019.03.01 21:59:20 INFO  web[][o.s.p.StopWatcher] Stopping process
2019.03.01 21:59:23 INFO  app[][o.s.a.SchedulerImpl] Process [web] is stopped
2019.03.01 21:59:24 INFO  app[][o.s.a.SchedulerImpl] Process [es] is stopped
2019.03.01 21:59:24 INFO  app[][o.s.a.SchedulerImpl] SonarQube is stopped
2019.03.01 21:59:24 WARN  app[][o.s.a.p.AbstractProcessMonitor] Process exited with exit value [es]: 143 
```
After that the server stops

- I'm already using the last scanner version (3.3.0.1492)
- I'm using the php scanner plugin, installed using the dashboard and going into marketplaces
- I'm using a different port (7500 instead of 9000) because I use php-fpm, but I've changed correctly on the sonnar-scanner.properties file:

```
#----- Default SonarQube server
sonar.host.url=http://sonarqube:7500

```
In a nutshell, the connection between the scanner (also docker) and sonarqube server is working perfectly but for some reason in the middle of the analysis this error occur.

I'd appreciate any help to direct me to a fix.
**PS: My host is a Mac OS High Sierra**

**Comment by simonbrandhof** [comment]
Hi, what's the version of SonarQube?

**Comment by pierre-guillot-sonarsource** [comment]
I'm closing this since there has been no update for a while. Please retry with the latest SQ version and create a new issue if needed.
